downstream/modules/automation-hub/proc-configure-content-signing-on-pah.adoc

```diff
@@ -0,0 +1,73 @@
+[id="proc-configure-content-signing-on-pah_{context}"]
+
+= Configuring content signing on private automation hub
+
+To successfully sign and publish {CertifiedName}, you must configure {PrivateHubName} for signing.
+
+.Prerequisites
+
+* Your GnuPG key pairs have been securely set up and managed by your organization.
+* Your public/private key pair has proper access for configuring content signing on {PrivateHubName}.
+
+.Procedure
+
+. Create a signing script that only accepts a filename.
++
+[NOTE]
+====
+This script will act as the signing service and must generate an ascii-armored detached `gpg` signature for that file using the key specified through the `PULP_SIGNING_KEY_FINGERPRINT` environment variable.
+====
++
+The script should then print out a JSON structure with the following format.
++
+----
+{"file": "filename", "signature": "filename.asc"}
+----
++
+All the file names are relative paths inside the current working directory. The filename must remain the same for the detached signature, as shown.
++
+The following example shows a script that produces signatures for content:
++
+[source,shell]
+----
+#!/usr/bin/env bash
+
+FILE_PATH=$1
+SIGNATURE_PATH="$1.asc"
+
+ADMIN_ID="$PULP_SIGNING_KEY_FINGERPRINT"
+PASSWORD="password"
+
+# Create a detached signature
+gpg --quiet --batch --pinentry-mode loopback --yes --passphrase \
+   $PASSWORD --homedir ~/.gnupg/ --detach-sign --default-key $ADMIN_ID \
+   --armor --output $SIGNATURE_PATH $FILE_PATH
+
+# Check the exit status
+STATUS=$?
+if [ $STATUS -eq 0 ]; then
+   echo {\"file\": \"$FILE_PATH\", \"signature\": \"$SIGNATURE_PATH\"}
+else
+   exit $STATUS
+fi
+----
+
++
+After you deploy a {PrivateHubName} with signing enabled to your {PlatformNameShort} cluster, new UI additions display when you interact with collections.
+
+. Review the output from the installer that begins with `automationhub_*``.
```

Comment:
This is from the installer inventory file which is what users use to setup their AAP cluster, not the output from installer

Basically, you will have to configure this before installing AAP or Private automation hub, to ensure signing is enabled on your server

Reply:
@anshulbehl - should this step be removed? How do we obtain the output from the installer?

Reply:
We don't need to obtain the output from installer, 
Instead of line 58 , we can just say
"Review the AAP installer inventory file for options that begins with `automationhub_*``. In order to configure your private automation hub with signing, you will have to set these options according to your deployment needs"

Reply:
Updated, per request.